website/public/js/services/userServices.js

```diff
@@ -201,6 +198,16 @@ angular.module('habitrpg')
           })
         },
 
+        clearNewMessages: function () {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/mark-pms-read',
+          })
+          .then(function (response) {
+            sync();
```

Comment:
maybe this should be a common op so we don't have to sync the whole user but just use the same code on the client and on the server?